pkg/sysinfo/sysinfo.go

```diff
@@ -3,23 +3,43 @@ package sysinfo
 // SysInfo stores information about which features a kernel supports.
 // TODO Windows: Factor out platform specific capabilities.
 type SysInfo struct {
+	// Wether the kernel supports AppArmor or not
 	AppArmor bool
+
 	*cgroupMemInfo
-	*cgroupCpuInfo
-	IPv4ForwardingDisabled        bool
-	BridgeNfCallIptablesDisabled  bool
-	BridgeNfCallIp6tablesDisabled bool
-	CgroupDevicesEnabled          bool
+	*cgroupCPUInfo
+
+	// Whether support IPv4 forwarding or not, if this was disabled, networking will not work
+	IPv4ForwardingDisabled bool
+
+	// Whether support bridge-nf-call-iptables or not
+	BridgeNfCallIptablesDisabled bool
+
+	// Whether support bridge-nf-call-ip6tables or not
+	BridgeNfCallIP6tablesDisabled bool
+
+	// Whether the cgroup has the mountpoint of "devices" or not
+	CgroupDevicesEnabled bool
 }
 
 type cgroupMemInfo struct {
-	MemoryLimit      bool
-	SwapLimit        bool
-	OomKillDisable   bool
+	// Whether support memory limit or not
+	MemoryLimit bool
+
+	// Whether support swap limit or not
+	SwapLimit bool
+
+	// Whether disable the OOM-Killer or not
+	OomKillDisable bool
+
+	// Whether support memory swappiness or not
 	MemorySwappiness bool
 }
 
-type cgroupCpuInfo struct {
-	CpuCfsPeriod bool
-	CpuCfsQuota  bool
+type cgroupCPUInfo struct {
+	// Whether support CPU CFS(Completely Fair Scheduler) period or not
+	CPUCfsPeriod bool
+
+	// Whether support CPU CFS(Completely Fair Scheduler) quota or not
+	CPUCfsQuota bool
```

Comment:
These fields in structure don't like functions that we can  easy to describe them with "FunctionA doing ..." . But if you can give me an example, I'd like to update